docs/index.rst

```diff
@@ -316,6 +316,20 @@ Configuration Options
   * Default: 60000
   * Importance: low
 
+``timestamp.delay.interval.ms``
+  How long to wait after a row with certain timestamp appears before we include it in the result. You may choose to add some delay to allow transactions with earlier timestamp to complete. The first execution will fetch all available records (i.e. starting at timestamp 0) until current time minus the delay. Every following execution will get data from the last time we fetched until current time minus the delay.
+
+  * Type: long
+  * Default: 0
+  * Importance: low
```

Comment:
Hmm, I don't think this was caught during the review when this option was added -- this was an oversight in the initial implementation and is probably pretty important. Should we raise the importance level here (at least to medium, maybe to high)?